README.md

```diff
@@ -16,10 +16,13 @@
 ```yaml
 name: Latest blog post workflow
 on:
-  schedule:
-    # Runs every hour
-    - cron: '0 * * * *'
-
+  schedule: # Run workflow automatically
+    - cron: '0 * * * *' # Runs every hour, on the hour
+  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the Github Actions Workflow page directly
+    inputs:
+      git-ref:
```

Comment:
I'm not sure of the usefulness of this actually. Maybe `inputs` can be removed altogether.

Reply:
Hum, haven't found any other way to avoid providing at least one param, so it looks like the best solution.

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/